pubmed_dl/bin/pubmed-dl

```diff
@@ -39,9 +39,12 @@ def main(start_date: str, end_date: str, request_file: str):
     output_filepath.parents[0].mkdir(parents=True, exist_ok=True)
     pmids = get_list_pmid(start_date, end_date)
     with open(output_filepath, "w") as f:
-        for batch in uids_to_docs(pmids):
-            for doc in batch:
-                f.write(f"{json.dumps(doc)}\n")
+        with typer.progressbar(length = len(pmids), label = "Processing") as progress:
+            for batch in uids_to_docs(pmids):
+                    for doc in batch:
+                        f.write(f"{json.dumps(doc)}\n")
+                        progress.update(1)
+                    print("")
```

Comment:
What's this for?

Reply:
The print is because the logging statements were being printed on the same line of the progress bar.
This made it look structured. 
I'll go ahead and merge.

Reply:
Doesn't this cause the progress bar to show up a bunch of times though? Once after every print?

Reply:
Yes the way it works, there are multiple progress bars but you will be able to see the completion rate after each 1000 records are read

Reply:
The way it is structured now, we are printing logging information after every batch of 1000 so, if we decide not to print that then we can have one progress bar. I was only able to set it up like this.